### ISSUE TITLE
Proposed changes to Wallet section

### ISSUE THREAD

**Issue by esot321c** [ergoplatform/ergoweb#116]
### A few notes for the Ergo Wallets section: 

1. It's difficult to find. If I go to the Ergo landing page, the only way to find wallets it the "Get Erg" button, then the "Store It" link. This is not obvious. The Ethereum website, the very first menu item says "use ethereum" and at the top of the list is "find wallet". 
2. If I don't realize the "Store it" link will lead to a wallet (I didn't. I had to be told in the chat that it was a link at the top of the page), I have to scroll halfway down the "Get erg" page to find the wallets section. 
3. When I get to the wallets section, if I haven't given up and gone back to BSC, the wallet list doesn't explain the priority, or which wallet a user should choose at first glance. In my opinion, the node should not be at the top. It is not user friendly. Any end user will not use the node, they will want to either use the mobile wallet, Nautlius, or SafeW. These are the only wallets currently supported by Ergo dApps, and so they should be the wallets recommended first to end users. 

### Proposed solutions:  

1. There should be a link on the landing page above the fold that uses the word "Wallet" and goes directly to a wallet summary section that is prioritized for end users. It can include node details, paper wallets, and other info, but the wallets that are useful for dapps such as nautilus, safew, and the mobile wallet should be prioritized. 
2. The wallet section would benefit from having its own page, and not be below the mining section on the Get Erg page. If the get erg page needs to contain mining, wallets, and exchanges, then the order should be wallets first, then exchanges, then mining. 
3. The wallet section should give users more info about each wallet. It could be an accordion that opens when you click on each wallet, which provides a summary of important features and considerations, plus links directly to the place where an end user can get the wallet. For example:

<img width="778" alt="image" src="https://user-images.githubusercontent.com/86281023/227365940-1316fa7a-bb34-4f63-b23a-6a667289e819.png">

At the bottom, it has a button that says "Check out MyCrypto". I would suggest for wallets like nautilus, it should also include the firefox and google play links, since the only current link to nautilus is a github link. An end user will arrive at github and immediately give up. They wont get down the to the readme and see that there are firefox and google play links there. 

### Additional feature suggestion: 
At the top of the wallet section, users could be asked what they want to do. It could be a select box or checkboxes which say: 

I want to (_use dapps, run a node, use a hardware wallet, have cold storage_)

One thing to note is that the mobile wallet does work with some dapps but not all. Maybe it could show up when people say "I want to use dapps" but have an asterisk that says "some dapps support mobile wallet" or something like that.

**Comment by FlyingPig69** [comment]
I agree with the comments that the current page is not particularily userfriendly for a total newbie to your community.
There are 3 types of users that will use he website:

1. Miners
2. Developers who want understand the ecosystem.
3. Users who just discovered ergo and want to buy and/or learn more


Of these 3 personas Miners and Developers will quickly be able to find what they need, they are incentivized to find the right info and will poke around unil they find what they need.

The 3rd persona, ecosystem users are the hardes to convince to stay. They migh have googled ergo or come across it on social media. You have 10 seconds to grab their attention before they leave!
Put yourself in their shoes, what do they want to achieve?

The majority want to find a wallet and/or learn where to buy. Make that stand out in a clear an obvious way.
Once they have their wallet/erg they want to learn about the dapps that are available, make that stand out and easy to find.

I totally get that here might not be a desire or resources available to do this right now, but if at least the top banner links could be redone to be more user friendly that would go a loooong way.


<img width="642" alt="banner" src="https://user-images.githubusercontent.com/124304224/227385994-2f63b97c-36cd-4e30-a43d-4fc406bdbe70.png">

**Comment by glasgowm148** [comment]
Thanks for opening the issue here! Sorry for the delay. 


1. User Journey

- Agree changing to 'Store It' -> 'Wallets' could help, I've got this one changed locally already and can push soon (battling with shite net!)
- Not sure it should be on it's own page unless we fleshed it out, nearer to the top might be better. Will speak to torto and see what he thinks.   
- Submenu is a good idea and easy enough to implement when @deadit is available


2. Wallet Section


- Agree about the node, I will change to 'Server' and move to the bottom. 
- Don't want to play favourites with the priority. ergcube has a good system, maybe we could replicate that or link there
- Paper Wallet is good for gifts (this could maybe be emphasised with a banner or a filter as you've said). 
- Minotaur has some unique features (like multisig) and already has a few dApps integrated so wouldn't want to exclude it. 
- Will look into an accordion hover.
